api/v1alpha3/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
are you using go 1.17?

Reply:
please use go 1.16.x for now

Reply:
still adding those on `go version go1.16.8 linux/amd64`

Reply:
I have reverted those manually.